core/domain/feedback_services.py

```diff
@@ -291,12 +308,39 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_email_task(user_id)
 
 
-def send_feedback_message_email(exploration_id, thread_id, message_id):
+def update_feedback_email_retries(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+    time = (datetime.datetime.utcnow() - model.created_on).seconds
+
+    if time > feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS:
+        model.retries += 1
+        model.put()
+
+
+def delete_feedback_message_references(user_id, references_length):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+
+    if references_length == len(model.feedback_message_references):
+        model.delete()
+    else:
+        message_references = (
+            model.feedback_message_references[references_length:])
+        model.delete()
```

Comment:
Actually, existing model has created_on property set to the hour before (since it is being processed hour after created). So if we modify existing model then next time when this existing model is processed, we will perceive it as a retry because of time difference. So creating new model solves this issue.